webdriver/tests/bidi/browsing_context/create/invalid.py

```diff
@@ -56,4 +56,5 @@ async def test_params_type_invalid_value(bidi_session, value):
 @pytest.mark.parametrize("value", [None, '', 42, {}, []])
 async def test_params_background_invalid_type(bidi_session, value):
     with pytest.raises(error.InvalidArgumentException):
-        await bidi_session.browsing_context.create(type_hint="tab", background = value)
+        new_context = await bidi_session.browsing_context.create(type_hint="tab", background=value)
+        await bidi_session.browsing_context.close(context=new_context["context"])
```

Comment:
hmmm... shouldn't `create` command fail and do not create a new tab? I think there shouldn't be a need to close the tab in this case 🤔

Reply:
yes, but if the implementation is wrong (e.g., current impls ignore the background flag), then it might create a context

Reply:
(here, I am trying to troubleshoot the chrome stability job but it does not look like the failure is actually caused by my change)

Reply:
my suspicion was that the context gets created somehow and leaks to the other tests (although I am not sure it the browser instance is restarted between the tests)